pybio/core/datasets/base.py

```diff
@@ -0,0 +1,34 @@
+import collections
+from typing import Dict, Optional, OrderedDict, Sequence, Tuple
```

Comment:
Use Dict instead of OrderedDict

Reply:
I went with MutableMapping instead, but seem to have forgotten it here... I'll fix it

Reply:
I don't know if MutableMapping implies ordering, while python Dict does.